sv_pipeline/exome/load_data.py

```diff
@@ -450,6 +469,12 @@ def main():
     if not es_password:
         es_password = getpass(prompt='Enter ES password: ')
 
+    sample_col_configs = NEW_JOINT_CALL_SAMPLE_COLS if args.is_new_joint_call else MERGING_CALL_SAMPLE_COLS
+    COL_CONFIGS.update(sample_col_configs)
```

Comment:
you can't update constants conditionally, its really unstable because you assume a constant is whatever it is defined as but in fact the constant could be one of many things depending on what calls were made to this function. You need to define 2 different configs at the top level, `COL_CONFIGS` and `JOINT_CALLED_COL_CONFIGS` and then conditionally use the correct configuration